backend/schemas/source/excel/libs/Sheets.libsonnet

```diff
@@ -23,12 +30,30 @@ local y_or_n_range = open_range {
   range_name: 'Example YorN range',
 };
 
+// TODO: import from audits.fixtures.excel
```

Comment:
The "schemas" directory is set up a bit like its own project, which makes importing from fixtures a little complicated. I tested out a couple of options, but none seemed great so I just copypasta'd for now.